client/engine/engine.go

```diff
@@ -184,6 +191,61 @@ func (e *Engine) handleProposal(proposal consensus_channel.Proposal) (EngineEven
 	return e.attemptProgress(obj)
 }
 
+// updateObjective handles updating and cranking the objective and dispatching any side effects
+func (e *Engine) updateObjective(objective protocols.Objective, event protocols.ObjectiveEvent) (EngineEvent, error) {
+	changeEvent := EngineEvent{}
```

Comment:
`changeEvent` was based on `EngineChangeEvent`. We should rename.

Reply:
Done in [d462846](https://github.com/statechannels/go-nitro/pull/826/commits/d4628466ade5a6fbbe8d742197034b197e57100e)